ereolen_support/ereolen_support.form.inc

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * @file
+ * Add eReolen support form.
+ */
+
+include_once 'ereolen_support.helper.inc';
+
+/**
+ * Create frontend form.
+ *
+ * @param array $form
+ *   The form to create.
+ * @param array $form_state
+ *   The state of the form.
+ *
+ * @return mixed
+ */
+function ereolen_support_form($form, &$form_state) {
+  global $user;
```

Comment:
The function call that uses the user object always returns NULL. So why have the user in this function?

Reply:
This file has been changed considerably